GiveNGo/GiveNGo
# Integrate Kitten UI in application

## Description:
- This change integrates Kitten UI into the application. Dependencies have been installed for this typescript and react native styling library.

## Changes I Made:
- [X] Create index.ts file for main entry into application. This file renders the top level App component.
- [X] In `App.tsx`, import Kitten UI components, remove react native components & styling, and add application provider which will wrap Routes component.
- [X] Create blank Routes.tsx file. 
- [X] In `package.json, package-lock.json, and yarn.lock`, install dependencies for Kitten UI. 

## How to Test:
1. npm install
2. expo start
3. Scan the QR code with your phone to test the application
4. The application should render. The filler text we added will render at the very top of your phone's screen (overlapped by battery indicator/status indicators from your phone.) A safe view port will need to be added in a later PR.
